Week-5/Exercise-5.4/calculator.js

```diff
@@ -0,0 +1,12 @@
+const mathOperations = {
+  sum: function (a, b) {
```

Comment:
why do we still not use arrow functions?

Reply:
Updated, Please check